lang/src/org/partiql/lang/ast/AstDeserialization.kt

```diff
@@ -107,6 +108,7 @@ private enum class NodeTag(val definition: TagDefinition) {
     CREATE(TagDefinition("create", AstVersion.V0, 2, 2)),
     DROP_TABLE(TagDefinition("drop_table", 1, 1)),
     DROP_INDEX(TagDefinition("drop_index", 2, 2)),
+    UNDROP(TagDefinition("undrop", AstVersion.V0, 2, 2)),
```

Comment:
I think we need to re-review the changes in this file more closely--we were trying for a while to leave the `V0` AST entirely unchanged.  This applies to both `AstSerialization.kt` and `AstDeserialization.kt`.  Some of these changes may need to be reverted so as to not break legacy clients that depend on the `V0` AST.

Reply:
Thanks for the catch. `UNDROP` should not have been included. Removed in the latest commit.